charts/scalardb-mcp-server/templates/scalardb-mcp-server/deployment.yaml

```
@@ -0,0 +1,98 @@
+kind: Deployment
+metadata:
+  name: {{ include "scalardb-mcp-server.fullname" . }}
+  namespace: {{ .Release.Namespace }}
+  labels:
+    {{- include "scalardb-mcp-server.labels" . | nindent 4 }}
+spec:
+  {{- if gt (int .Values.scalardbMcpServer.replicaCount) 1 }}
+  {{- fail "Multiple replicas is currently not supported. Please set replicaCount to 1." }}
+  {{- end }}
+  replicas: {{ .Values.scalardbMcpServer.replicaCount }}
+  strategy:
+    type: Recreate
+  selector:
+    matchLabels:
+      {{- include "scalardb-mcp-server.selectorLabels" . | nindent 6 }}
+  template:
+    metadata:
+      annotations:
+        checksum/config: {{ include (print $.Template.BasePath "/scalardb-mcp-server/configmap.yaml") . | sha256sum }}
+        {{- with .Values.scalardbMcpServer.podAnnotations }}
+        {{- toYaml . | nindent 8 }}
+        {{- end }}
+      labels:
+        {{- include "scalardb-mcp-server.selectorLabels" . | nindent 8 }}
+        {{- with .Values.scalardbMcpServer.podLabels }}
+        {{- toYaml . | nindent 8 }}
+        {{- end }}
+    spec:
+      restartPolicy: Always
+      serviceAccountName: {{ include "scalardb-mcp-server.serviceAccountName" . }}
+      automountServiceAccountToken: {{ .Values.scalardbMcpServer.serviceAccount.automountServiceAccountToken }}
+      {{- with .Values.scalardbMcpServer.imagePullSecrets }}
+      imagePullSecrets:
+        {{- toYaml . | nindent 8 }}
+      {{- end }}
+      securityContext:
+        {{- toYaml .Values.scalardbMcpServer.podSecurityContext | nindent 8 }}
+      containers:
+        - name: {{ .Chart.Name }}
+          image: "{{ .Values.scalardbMcpServer.image.repository }}:{{ .Values.scalardbMcpServer.image.tag | default .Chart.AppVersion }}"
+          imagePullPolicy: {{ .Values.scalardbMcpServer.image.pullPolicy }}
+          securityContext:
+            {{- toYaml .Values.scalardbMcpServer.securityContext | nindent 12 }}
+          ports:
+            - name: http
+              containerPort: {{ .Values.scalardbMcpServer.service.targetPort }}
+              protocol: {{ .Values.scalardbMcpServer.service.protocol }}
+          {{- with .Values.scalardbMcpServer.env }}
+          env:
+            {{- toYaml . | nindent 12 }}
+          {{- end }}
+          livenessProbe:
+            httpGet:
+              path: /actuator/health/liveness
+              port: http
+            initialDelaySeconds: 30
+            periodSeconds: 10
+            failureThreshold: 3
+          readinessProbe:
+            httpGet:
+              path: /actuator/health/readiness
+              port: http
+            initialDelaySeconds: 10
+            periodSeconds: 5
+            failureThreshold: 3
+          volumeMounts:
+            - name: application-properties-volume
+              mountPath: /scalardb-mcp-server/application.properties
+              subPath: application.properties
+            {{- if .Values.scalardbMcpServer.persistence.enabled }}
+            - name: data-volume
```

```suggestion
            - name: session-data-volume
```
I feel it's a bit difficult to know what data is. If we can simply represent what this MCP Server stores, I think it would be better to include such information in the name, for example, `session-data-volume`.
(If I remember correctly, you explained that this data volume is used for storing session data in the previous meeting. Please correct me if I miss something. 🙇)